images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -66,4 +66,12 @@ Describe "Android SDK" {
     It "Additional tool <additionalToolVersion> is installed" -TestCases $additionalToolsTestCases {
         "$installedPackages" | Should -Match $additionalToolVersion
     }
-}
\ No newline at end of file
+
+    It "LTS NDK is installed" {
+        "$androidInstalledPackages" | Should -Match $androidToolset.ndk.lts
```

Comment:
I think we should do a more strict check here, something like this:
```
... | Should -Match "ndk;$androidToolset.ndk.lts"
```

Reply:
Done, thanks!